nickhartdev/Overlook
# Landing pages

###### What's this PR do?
- Adds a non styled user landing page that shows all info required in spec
- Adds a non styled manager landing page that shows all info required in spec
- Adds all necessary functions to User and Hotel classes for calculating data requested in spec
- Adds all domUpdates functions for displaying information requested in spec
- Refactors class structure to more efficiently handle fetch requests and return 
- Adds all corresponding tests for functions written so far
###### Where should the reviewer start?
dataFetcher.js, to see how the info is being pulled from the server. I'd then recommend going over the tests for the User and Hotel classes to see how they're used.
###### How should this be manually tested?
Log in as different users and as a manager to see how information is currently pulled and displayed.
###### Any background context you want to provide?
The pages are currently not styled, as css is my lowest priority right now. Going forward my pages are going to be remain unstyled (while just focusing on how I want to structure the page with my HTML) and then styling will be layered in at the end. I'm currently just focused on functionality.
###### What are the relevant tickets?
Closes #4, closes #5, closes #3, closes #8, closes #9, closes #10, closes #11, closes #12
###### Questions?
I've left notes at places in the code where I have more specific questions!
